pkg/controller/toolchainenabler/toolchainenabler_controller.go

```diff
@@ -190,15 +192,17 @@ func (r *ReconcileToolChainEnabler) ensureClusterRoleBinding(tce *codereadyv1alp
 	if err := controllerutil.SetControllerReference(tce, crb, r.scheme); err != nil {
 		return err
 	}
-	_, err := r.client.GetClusterRoleBinding(CRBName)
-	if err != nil && errors.IsNotFound(err) {
-		log.Info("Adding `self-provisioner` cluster role to ", "Service Account", saName)
-		if err := r.client.CreateClusterRoleBinding(crb); err != nil {
-			return err
-		}
+	if _, err := r.client.GetClusterRoleBinding(CRBName); err != nil {
+		if errors.IsNotFound(err) {
+			log.Info("Adding `self-provisioner` cluster role to ", "Service Account", saName)
```

Comment:
I suggest you use a backtick to encapsulate the whole string so you're free to use more comment quotes inside:

```suggestion
			log.Info(`Adding "self-provisioner" cluster role to `, "Service Account", saName)
```

or 


```suggestion
			log.Info(`Adding 'self-provisioner' cluster role to `, "Service Account", saName)
```

or 


```suggestion
			log.Info("Adding 'self-provisioner' cluster role to ", "Service Account", saName)
```

But please don't use backticks as quotes in strings.